src/vs/workbench/api/browser/mainThreadWorkspace.ts

```diff
@@ -123,11 +123,11 @@ export class MainThreadWorkspace implements MainThreadWorkspaceShape {
 
 	// --- search ---
 
-	$startFileSearch(includePattern: string | null, _includeFolder: UriComponents | null, excludePatternOrDisregardExcludes: string | false | null, maxResults: number | null, token: CancellationToken): Promise<UriComponents[] | null> {
+	$startFileSearch(includePattern: string | undefined, _includeFolder: UriComponents | undefined, excludePatternOrDisregardExcludes: string | false | undefined, maxResults: number | undefined, token: CancellationToken): Promise<UriComponents[] | undefined> {
```

Comment:
Cosmetic changes? There is debate in the team if `undefined` or `null` should be used and I don't wanna end up bike shedding. I prefer `undefined` but I will not accept a PR with such changes unless a new version of TS mandates it

Reply:
Uhhh, some of these APIs flip-flopped weather they used `null` or `undefined` while I was writing the PR (vscode master was briefly broken, even) and I just did my best to reconcile at each step. IIRC this one I swapped to stifle a mismatch triggered by a change made in the vscode `master` branch somewhere else.